app/EventBusProvider.php

```diff
@@ -0,0 +1,52 @@
+<?php declare(strict_types=1);
+
+namespace CultuurNet\UDB3\SearchService;
+
+use Broadway\EventHandling\EventBusInterface;
+use CultuurNet\UDB3\Search\SimpleEventBus;
+
+class EventBusProvider extends BaseServiceProvider
+{
+
+    protected $provides = [
+        EventBusInterface::class,
+    ];
+
+    /**
+     * Use the register method to register items with the container via the
+     * protected $this->leagueContainer property or the `getLeagueContainer` method
+     * from the ContainerAwareTrait.
+     *
+     * @return void
+     */
+    public function register()
+    {
+        $this->add(
+            EventBusInterface::class,
+            function () {
+                $bus = new SimpleEventBus();
+
+                $bus->beforeFirstPublication(function (EventBusInterface $eventBus) {
+                    $subscribers = $this->get('event_bus_subscribers');
+
+                    if (!(is_null($this->parameter('config.event_bus'))) &&
+                        !(is_null($this->parameter('config.event_bus.subscribers')))) {
+
+                        $subscriberIds = $this->parameter('config.event_bus.subscribers');
+                        $subscribers = [];
+                        foreach ($subscriberIds as $subscriberServiceId) {
+                            $subscribers[] = $subscriberServiceId;
```

Comment:
```suggestion
                            $subscribers[] = $this->get($subscriberServiceId);
```
The ids are just strings from the config file so we should fetch the actual services from the service container before subscribing them.